examples/svelte-apollo/src/apollo.js

```diff
@@ -0,0 +1,56 @@
+import ApolloClient from "apollo-client";
+import { NhostClient } from '@nhost/nhost-js';
+import { InMemoryCache } from "apollo-cache-inmemory";
+import { WebSocketLink } from "apollo-link-ws";
+import { split } from "apollo-link";
+import { HttpLink } from "apollo-link-http";
+import { getMainDefinition } from "apollo-utilities";
+require('dotenv').config()
+
+const nhost = new NhostClient({
+  backendUrl: process.env.backend_uri,
+});
+
+const accessToken = nhost.auth.getAccessToken()
+
+const headers = {
+  'content-type': 'application/json',
+  'x-hasura-admin-secret': accessToken
+};
+
+const getHeaders = () => {
+  return headers;
```

Comment:
The access token is updated every 15 minutes so it's better to get the access token inside this `getHeaders()` function so the access token is always up-to-date for all GraphQL requests.

Reply:
This is left.yet, working on this part.